<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicSourceUpdateWidget</class>
 <widget class="QDialog" name="MusicSourceUpdateWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>360</width>
     <height>260</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>软件升级精灵</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>358</x>
     <y>22</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundMask" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>45</y>
     <width>360</width>
     <height>235</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(255,255,255,200)</string>
   </property>
   <widget class="QLabel" name="mainLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>110</width>
      <height>215</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../musicQrc/MusicPlayerShare.qrc">:/share/upgradeMain</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>119</x>
      <y>10</y>
      <width>231</width>
      <height>215</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="titleLable_F">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>1</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>天天酷音已经有新版本退出</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="contextLable_F">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>211</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>版本: 2.2.8.8    2015.12.07
1.增加高品质选项和无损
2.用户自动登录问题修复
3.优化mv播放器
4.修复mv进度条拖放不正确问题


</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="iconLabel_F">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>170</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="pixmap">
       <pixmap resource="../../musicQrc/MusicPlayerShare.qrc">:/share/upgrade</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="upgradeButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>180</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{border-radius:5px;background-color:#80B7F1;color:white;font-size:13px;font-weight:bold;}</string>
      </property>
      <property name="text">
       <string>更新</string>
      </property>
     </widget>
     <widget class="QLabel" name="upgradeFailedLabel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>183</y>
        <width>55</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;font color=#0000FF&gt;&lt;u&gt;升级失败?&lt;/u&gt;&lt;/font&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page2">
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>221</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>当前的文档</string>
      </property>
      <widget class="QProgressBar" name="downProgressBar">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>75</y>
         <width>200</width>
         <height>25</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="fileSizeLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>25</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>文档大小: 1802.5KB</string>
       </property>
      </widget>
      <widget class="QLabel" name="speedLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>传送速率: 801KB/s</string>
       </property>
      </widget>
      <widget class="QLabel" name="timeLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>55</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>估计剩余时间: 0:00:02</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>221</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>全部的文档</string>
      </property>
      <widget class="QProgressBar" name="downProgressBarAL">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>45</y>
         <width>200</width>
         <height>25</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="downloadingLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>22</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>当前正在下载: 1/1</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <zorder>background</zorder>
  <zorder>backgroundMask</zorder>
  <zorder>topTitleName</zorder>
  <zorder>topTitleCloseButton</zorder>
 </widget>
 <resources>
  <include location="../../musicQrc/MusicPlayerShare.qrc"/>
  <include location="../../musicQrc/MusicPlayerShare.qrc"/>
 </resources>
 <connections/>
</ui>
